VadeSecure/vadesecure_modules/models.py

```diff
@@ -15,3 +16,12 @@ class VadeSecureTriggerConfiguration(BaseModel):
 
     pagination_limit: int = Field(100, description="Limit of records to fetch from the API per request")
     rate_limit: int = Field(10, description="Limit of requests per second to VadeSecure API")
+
+
+class VadeSecureConnectorConfiguration(DefaultConnectorConfiguration):
+    tenant_id: str = Field(..., description="Identifier of your 365 Tenant")
+    frequency: int = Field(..., description="Batch frequency in seconds")
+    chunk_size: int = Field(10000, description="The max size of chunks for the batch processing")
+
+    pagination_limit: int = Field(100, description="Limit of records to fetch from the API per request")
+    rate_limit: int = Field(10, description="Limit of requests per second to VadeSecure API")
```

Comment:
As the class has the same properties that the one above, can you inherit this class from the upper one?
```suggestion
class VadeSecureConnectorConfiguration(DefaultConnectorConfiguration, VadeSecureTriggerConfiguration):
    pass
```